### PR TITLE
Change port of socks proxy to non-standard

### PR DESCRIPTION
**Pull request by maciejgrzybek**
This is to unify access to Hive through socks proxy from inside and outside of docker.

---

Partial work is done here by David: https://github.com/prestodb/presto/pull/6274#pullrequestreview-2943884
Mind my comments in that PR, please.

Travis build in prestodb/presto using snapshot images produced by this PR: https://travis-ci.org/maciejgrzybek/presto/builds/165354678

### CODE REVIEW

**Comment by ebd2** [comment]
LGTM

File: teradatalabs/cdh5-hive/Dockerfile [hunk 1]
```diff
@@ -59,6 +59,6 @@ EXPOSE 8032 8030 8031 8033 8088 8040 8042 8041 10020 19888
 EXPOSE 10000 9083
 
 # SOCKS PORT
-EXPOSE 1080
+EXPOSE 1180
```
Comment (ArturGajowy) [inline teradatalabs/cdh5-hive/Dockerfile:62]: I think ur missing one more dockerfile for this change

Comment (ArturGajowy) [reply teradatalabs/cdh5-hive/Dockerfile:62]: Nah, dammit. It's me who didn't add that port in the first place. Can you add it to hdp2.5-hive/Dockerfile? :)

**Comment by ArturGajowy** [comment]
It seems the kerberos tests failed. Maybe that image was not rebuilt? Or the old version was used?

**Comment by maciejgrzybek** [comment]
I didn't change docker-compose.yml properly. Fixed and restarted build.

File: teradatalabs/cdh5-hive/Dockerfile (hunk 1 above)
Comment (maciejgrzybek) [reply teradatalabs/cdh5-hive/Dockerfile:62]: Will do!
I've seen that but thought it's not needed so I left it unchanged.

Comment (maciejgrzybek) [reply teradatalabs/cdh5-hive/Dockerfile:62]: @ArturGajowy done in 5a94c40.

**Comment by ArturGajowy** [comment]
It seems the're still failing, though... Thanks for the port addition! LGTM as soon as the build passes :)

**Comment by maciejgrzybek** [comment]
https://travis-ci.org/Teradata/presto/builds/165422982 passes with this image.

**Comment by maciejgrzybek** [comment]
Merged, thanks!

**Pull request merged**